README.md

```diff
@@ -4,6 +4,8 @@
 
 This plugin for Jenkins pulls Test Automation data from Dynatrace AppMon and displays it through charts and tables on the project and build level.
 
+Since version 2.1.0 plugin supports Jenkins Pipelines. Required configuration and usage examples are described [here](pipeline-steps-explained.md). Jenkins 2.0 and later is officially supported.
```

Comment:
Release 2.1.0 adds support for Jenkins Pipelines. Configuration and usage examples are described [here](pipeline-steps-explained.md).